.github/workflows/yum.yml

```diff
@@ -301,6 +301,7 @@ jobs:
           - "update-to-next-major-version.sh etc inactive"
           - "update-without-data-lost.sh v6 v7"
           - "update-without-data-lost.sh v7 v6"
+          - "builtin-plugins.sh"
```

Comment:
As take it into account for adding update-notifier test case in the future, 
obsolete-plugins.sh might be better to clarify.